packages/app-mobile/android/build.gradle

```diff
@@ -2,23 +2,23 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "33.0.0"
-        minSdkVersion = 21
+        buildToolsVersion = "34.0.0"
+        minSdkVersion = 23
```

Comment:
This increases the minimum supported Android version. [`minSdkVersion = 23` corresponds to Android 6](https://developer.android.com/tools/releases/platforms).